deploy/helm/elastic-agent/values.yaml

```diff
@@ -300,6 +300,15 @@ kubernetes:
       # -- system metric stream vars
       # @section -- 2 - Kubernetes integration
       vars: {}
+  event:
+    # -- enable [event](https://www.elastic.co/guide/en/beats/metricbeat/8.11/metricbeat-metricset-kubernetes-event.html)
+    # input
+    # @section -- 2 - Kubernetes integration
+    enabled: false
+    # -- event stream variables such as `hosts`, `period`, `bearer_token_file`,
+    # `ssl.verification_mode`, `condition`.
+    # @section -- 2 - Kubernetes integration
+    vars: {}
```

Comment:
we are in the process of choosing the right tool to automate our helm chart README generation. But until then please add what you have here in the values, under this https://github.com/elastic/elastic-agent/blob/b4943d89719e1337b27cb4302917297e4865ae6e/deploy/helm/elastic-agent/README.md?plain=1#L127 🙂